persistence/startup-folder/write-file-to-startup-folder.yml

```diff
@@ -12,9 +13,20 @@ rule:
     examples:
       - 07F7846BBCDA782E5639292AD93907EB:0x401040
   features:
-    - and:
-      - match: get startup folder
-      - or:
-        - match: copy file
-        - match: move file
-        - match: host-interaction/file-system/write
+    - or:
+      - and:
+        - match: get startup folder
+        - or:
+          - match: copy file
+          - match: move file
+          - match: host-interaction/file-system/write
+      - call:
+        - and:
+          - or:
+            - string: /Microsoft\\Windows\\Start Menu\\Programs\\Startup\\/i
+            - string: /Documents and Settings\\All Users\\Start Menu\\Programs\\Startup\\/i
+            - string: /WINNT\\Profiles\\All Users\\Start Menu\\Programs\\Startup\\/i
```

Comment:
let's move these to `get startup folder`?

Reply:
Side note: this rule could benefit from the sliding-window scope idea

Reply:
I am hesitant to move the paths to `get startup folder`, because it may result in many false positives. Currently, the paths are within the call scope. If we move the paths to `get startup folder`, it will be in the thread scope.

Reply:
Also, merge with `reference-startup-folder.yml`?

Reply:
hm, reference startup folder definitely also catches these so merging would make sense, we'll just have to see about the scopes

Reply:
@mr-tz see my latest commit. I think this is a clean fix.